uint8 request
## 0 requests all sensors,
## 1 requests avoidance sensors, dirt detectors, motor overcurrents
## 2 requests remote control, button state, distance traveled, heading change
## 3 requests battery stats
---
bool caster_down
bool left_wheel_down
bool right_wheel_down
bool lbump
bool rbump
bool wall
bool lcliff
bool lfcliff
bool rcliff
bool rfcliff
bool vwall
bool lmotoroverload
bool rmotoroverload
bool vacmotoroverload
bool sidemotoroverload
bool brushmotoroverload
uint8 ldirt
uint8 rdirt
uint8 remote
bool powerbutton
bool cleanbutton
bool spotbutton
bool maxbutton
int16 distance
float32 angle
uint8 chargestate
uint16 voltage
int16 current
int8 temperature
uint16 charge
uint16 capacity
